index.js

```diff
@@ -35,7 +35,9 @@ const checkPullRequestReviewModule = require('./lib/checkPullRequestReview');
 const newCodeOwnerModule = require('./lib/checkForNewCodeowner');
 const ciCheckModule = require('./lib/ciChecks');
 const periodicCheckModule = require('./lib/periodicChecks');
-
+const checkAfter24Hrs = require(
+  './lib/pingCodeOwner'
+);

```

Comment:
This functionality should be moved to the pull request review module.

Reply:
See https://github.com/oppia/oppiabot/pull/215#issuecomment-808814802

Reply:
Ok sir.